docs/getting-started-guides/kops.md

```diff
@@ -0,0 +1,152 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart shows you how to easily install a Kubernetes cluster on AWS.
+It uses a tool called [`kops`](https://github.com/kubernetes/kops).
+
+kops is an opinionated provisioning system:
+
+* Fully automated installation
+* Uses DNS to identify clusters
+* Self-healing: everything runs in Auto-Scaling Groups
+* Limited OS support (Debian preferred, Ubuntu 16.04 supported, early support for CentOS & RHEL)
+* High-Availability support
+* Can directly provision, or generate terraform manifests
+
+If your opinions differ from these you may prefer to build your own cluster using [kubeadm](kubeadm) as
+a building block.  kops builds on the kubeadm work.
+
+## Creating a cluster
+
+### (1/5) Install kops
+
+Download kops from the [releases page](https://github.com/kubernetes/kops/releases) (it is also easy to build from source):
+
+On MacOS:
+
+```
+wget https://github.com/kubernetes/kops/releases/download/v1.4.1/kops-darwin-amd64
+chmod +x kops-darwin-amd64
+mv kops-darwin-amd64 /usr/local/bin/kops
+```
+
+On Linux:
+
+```
+wget https://github.com/kubernetes/kops/releases/download/v1.4.1/kops-linux-amd64
+chmod +x kops-linux-amd64
+mv kops-linux-amd64 /usr/local/bin/kops
+```
+
+### (2/5) Create a route53 domain for your cluster
+
+kops uses DNS for discovery, both inside the cluster and so that you can reach the kubernetes API server
+from clients.
+
+kops has a strong opinion on the cluster name: it should be a valid DNS name.  By doing so you will
+no longer get your clusters confused, you can share clusters with your colleagues unambigiously,
+and you can reach them without relying on remembering an IP address. 
+
+You can, and probably should, use subdomains to divide your clusters.  As our example we will use
+`useast1.dev.example.com`.  The API server endpoint will then be `api.useast1.dev.example.com`.
+
+A Route53 hosted zone can serve subdomains.  Your hosted zone could be `useast1.dev.example.com`,
+but also `dev.example.com` or even `example.com`.  kops works with any of these, so typically
+you choose for organization reasons (e.g. you are allowed to create records under `dev.example.com`,
+but not under `example.com`).
+
+Let's assume you're using `dev.example.com` as your hosted zone.  You create that hosted zone using
+the [normal process](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html),
```

Comment:
Can this be done by the AWS CLI?  If so, that might be best as you assume that folks are using it in the next step.